### PR TITLE
Bug fix: FindAll() only if user is admin.

### PR DESCRIPTION
**Pull request by romain-ortega**
Before a simple user could be disconnected because he get an Unauthorized response from backend.
Now it call the backend only if the user is an administrator.

### CODE REVIEW

File: webapp/app/protected/users/route.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+
+  session: Ember.inject.service('session'),
```
Comment (willrnch) [inline webapp/app/protected/users/route.js]: Useless. Session is injected by default.

File: webapp/app/protected/users/route.js [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+  model() {
+    if (this.get('session.user.isAdmin')) {
+      return this.store.findAll('user', { reload: true });
```
Comment (willrnch) [inline webapp/app/protected/users/route.js]: You should do this in the `protected.users.index` route (`webapp/app/protected/users/index/route.js`).

File: webapp/app/protected/users/user/template.hbs [hunk 3]
```diff
@@ -1,77 +1,79 @@
 <div class="section">
-	<h4>
-		{{#if model.isAdmin }}
```
Comment (Costent) [inline webapp/app/protected/users/user/template.hbs]: You probably want to check the current connected user `session.user.isAdmin`. Not the displayed user

File: webapp/app/protected/users/user/template.hbs [hunk 4]
```diff
@@ -1,77 +1,79 @@
...
-			type="text"
-			confirmation=false
-			textInput=model.lastName
-			textInputPlaceholder="Last name"
-			onClose="changeLastName"
-		}}
-			{{model.lastName}}
-		{{/edit-text}}
+  <h4>
+    {{#if model.isAdmin }}
```
Comment (Costent) [inline webapp/app/protected/users/user/template.hbs]: You probably want to check the connected user (`session.user.isAdmin`). Not the displayed user.

File: webapp/app/protected/users/user/template.hbs [hunk 5]
```diff
@@ -1,77 +1,79 @@
...
 
-		{{#if model.isAdmin}}
-			<i class="material-icons va-top">verified_user</i>
-		{{/if}}
-	</h4>
-	<div class='content-wrapper'>
-		<table class="table user-table">
-			<tbody>
+    {{#if model.isAdmin}}
+      <i class="material-icons va-top">verified_user</i>
```
Comment (Costent) [inline webapp/app/protected/users/user/template.hbs]: `verified_user`? `admin` no?

**Pull request merged**